values.yaml

```diff
@@ -106,6 +106,9 @@ server:
               component: server
           topologyKey: kubernetes.io/hostname
 
+  # Extra annotations to attach to the server pods
+  annotations: {}
```

Comment:
For complicated inputs in the values.yaml file, we've standardized on using multi-line strings. For consistency, this value should use the same pattern. This would make the annotation definition work like this:
```
  annotations: |
    "my-annotation": "my-annotation-value"
    "foo": "bar"
```

These values are processed in the template files with `tpl` instead of `toYaml1`. A good recent example of this is #105.